subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/FileCacheBackedScriptClassCompiler.java

```diff
@@ -97,32 +101,66 @@ public FileCacheBackedScriptClassCompiler(CacheRepository cacheRepository, Scrip
         // Both caches can be closed directly after use because:
         // For 1, if the script changes or its compile classpath changes, a different directory will be used
         // For 2, if the script changes, a different cache is used. If the classpath changes, the cache is invalidated, but classes are remapped to 1. anyway so never directly used
-        PersistentCache remappedClassesCache = cacheRepository.cache("scripts-remapped/" + source.getClassName() + "/" + sourceHash + "/" + classpathHash)
-            .withDisplayName(dslId + " remapped class cache for " + sourceHash)
-            .withInitializer(new ProgressReportingInitializer(progressLoggerFactory, new RemapBuildScriptsAction<>(remapped, classpathHash, sourceHash, dslId, classLoader, operation, verifier, scriptBaseClass),
+        final PersistentCache cache = cacheRepository.cache("scripts/" + sourceHash + "/" + classpathHash)
```

Comment:
❓ With this change, shouldn't the script cache cleanup for CI be updated?

https://github.com/gradle/gradle/blob/b5aa8554d5da43d4267ce1d240aa7a98399f23eb/buildSrc/subprojects/cleanup/src/main/kotlin/org/gradle/gradlebuild/testing/integrationtests/cleanup/Cleanup.kt#L95